driver/layernorm_driver.hpp

```diff
@@ -24,7 +24,6 @@
  *
  *******************************************************************************/
 #include <miopen/miopen.h>
-#ifdef MIOPEN_BETA_API
```

Comment:
Please 
```C++
#define MIOPEN_BETA_API 1
```
prior any inclusion of miopen.h in our source code. This is to make beta declarations/definitions visible in the library sources.

Reply:
What you mean is leave MIOPEN_BETA_API as is and just define MIOPEN_BETA_API before miopen.h?
I understood that MIOPEN_BETA_API should only exist in miopen.h. https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2430#discussion_r1352807948

Reply:
@seungmanhan 
> What you mean is leave MIOPEN_BETA_API as is and just define MIOPEN_BETA_API before miopen.h?

No. The conditional (`#ifdef MIOPEN_BETA_API`) is correctly removed in this PR. But the PR misses definition of MIOPEN_BETA_API before `#include <miopen/miopen.h>`.

> https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2430#discussion_r1352807948

That comment says that _conditionals_ should be removed. But we need _definitions_ of MIOPEN_BETA_API because:

> This is to make beta declarations/definitions \<from miopen.h\> visible in the library sources.

Reply:
fixed this

Reply:
[resolved]

Reply:
@seungmanhan We actually do not need this, please look at https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2481#discussion_r1383825898